tests/post_training/model_scope.py

```diff
@@ -250,6 +250,7 @@
         },
         "backends": [BackendType.OV],
         "batch_size": 32,
+        "validation_batch_size": 1,  # Validation is slower with batch_size > 1
```

Comment:
Why do we have separate batch sizes for calibration & validation? If some models should be validated with the batch_size=1, they should be calibrated with the same batch.

Reply:
Make sense

Reply:
set bs=1 for swin